docker/production.yaml

```diff
@@ -0,0 +1,55 @@
+listen:
+  port: 9000
+
+# Correspond to your reverse proxy "listen" configuration
+webserver:
+  https: true
+  hostname: undefined
```

Comment:
Will the yaml parser consider this to be a string with the value "undefined" or the real undefined value? 🤔

Reply:
The latter I think. Got that from the [node-config documentation](https://github.com/lorenwest/node-config/wiki/Common-Usage#using-config-values), although I may have misunderstood it.

Reply:
It will set it as the undefined JavaScript object and fail configuration validation if the value is not overloaded by an environment variable.

Reply:
:+1: That's what I wanted.